### PR TITLE
💅Assorted polishes

### PR DESCRIPTION
**Pull request by jaronheard**
Probably not a good habit to start, but this PR incorporates a number of design changes and improvements that will keep CIVIC looking her best.

Component library:
 - Redesigned gradient scale component, per request from Cat.
 - Resized pie chart labels to improve fit in information hierarchy
 - Removed background from Collapsable and changed font to be more consistent with style guide
 - Changed non-standard colors in axis and grid of charts
 - Removed link underline to be consistent with style guide

Story collections:
 - content improvements
 - labeling improvements, including eliminating all overlapping labels from initial view
 - removed a non-standard card from housing (PDF embed)

**BEFORE**
------------------------------
image
**AFTER**
-------------------------------
image

### CODE REVIEW

File: packages/2018-local-elections/src/components/OutraisingYourOpponent/index.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ export class OutraisingYourOpponent extends React.Component {
           sortOrder="sortOrder"
           data={data}
           dataValueFormatter={percentage}
+          domain={{x: [0,0.6], y: [1,11]}}
```
Comment (DingoEatingFuzz) [inline packages/2018-local-elections/src/components/OutraisingYourOpponent/index.js]: This is very specific and not associated with the data...but I'll turn a blind eye 🙈

File: packages/2018-local-elections/src/components/RealTimeInformationOnPoliticalCampaigns/ContributorBreakdown.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const ContributorBreakdown = ({ contributors, loading }) => {
       loading={loading}
       data={data}
       title="Contributor breakdown"
-      subtitle="Breakdown of contributors..."
+      subtitle="Contributions reported to ORSTAR by category"
```
Comment (DingoEatingFuzz) [inline packages/2018-local-elections/src/components/RealTimeInformationOnPoliticalCampaigns/ContributorBreakdown.js]: This should "OreStar"

File: packages/2018-local-elections/src/components/RealTimeInformationOnPoliticalCampaigns/SpendingBreakdown.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ const SpendingBreakdown = ({ spending, loading }) => {
       xLabel="Category"
       yLabel="Spending"
       title="Spending breakdown"
-      subtitle="Breakdown of spending..."
+      subtitle="Spending reported to ORSTAR by category"
```
Comment (DingoEatingFuzz) [inline packages/2018-local-elections/src/components/RealTimeInformationOnPoliticalCampaigns/SpendingBreakdown.js]: OreStar again.

File: packages/2018-local-elections/src/components/OutraisingYourOpponent/index.js (hunk 1 above)
Comment (jaronheard) [reply packages/2018-local-elections/src/components/OutraisingYourOpponent/index.js]: 🙈

**Pull request merged**